apps/dashboard/app/(app)/settings/team/client.tsx

```diff
@@ -69,17 +69,15 @@ export default function TeamPageClient({ team }: { team: boolean }) {
 
   if (!team) {
     return (
-      <div className="w-full h-screen -mt-40 flex items-center justify-center">
-        <Empty>
-          <Empty.Title>Upgrade Your Plan to Add Team Members</Empty.Title>
-          <Empty.Description>You can try it out for free for 14 days.</Empty.Description>
-          <Empty.Actions>
-            <Link href="/settings/billing">
-              <Button>Upgrade</Button>
-            </Link>
-          </Empty.Actions>
-        </Empty>
-      </div>
+      <Empty className="xl:mt-96">
```

Comment:
Maybe we should center this no matter what screen this is, like `xl:mt-96`? We can go with flex or 

```
  position: absolute;
  top: 50%; right: 50%;
  transform: translate(50%,-50%);

```
something like this